server-monitor

```diff
@@ -4,51 +4,83 @@
             "Address": "ether1.org",
             "Port": 80
         }, {
-            "Name": "ethoFS Website",
+            "Name": "Main ethoFS Website",
             "Address": "ethofs.com",
             "Port": 80
         }, {
-            "Name": "Ether-1 Staking Website",
+            "Name": "Main Ether-1 Staking Website",
             "Address": "staking.ethofs.com",
             "Port": 80
         }, {
-            "Name": "Ether-1 Nodes Website",
+            "Name": "Main Ether-1 Nodes Website",
             "Address": "nodes.ether1.org",
             "Port": 80
 	}, {
-	    "Name": "Ether-1 Wallet Website",
+	    "Name": "Main Ether-1 Wallet Website",
             "Address": "wallet.ether1.org",
             "Port": 80
 	}, {
-            "Name": "Ether-1 Blocks Explorer Website",
+            "Name": "Main Ether-1 Secondary Explorer Website",
             "Address": "blocks.ether1.org",
             "Port": 80
 	}, {
-            "Name": "Ether-1 Explorer Website",
+            "Name": "Main Ether-1 Primary Explorer Website",
             "Address": "explorer.ether1.org",
             "Port": 80
 	}, {
-	    "Name": "Ether-1 Richlist Website",
+	    "Name": "Main Ether-1 Richlist Website",
             "Address": "richlist.ether1.org",
             "Port": 80
 	}, {
-	    "Name": "Ether-1 Stats Website",
+	    "Name": "Main Ether-1 Stats Website",
             "Address": "stats.ether1.org",
             "Port": 80
 	}, {
-	    "Name": "Ether-1 Documentation Website",
+	    "Name": "Main Ether-1 Documentation Website",
             "Address": "docs.ether1.org",
             "Port": 80
 	}, {
             "Name": "Main Xerom Website",
             "Address": "xerom.org",
             "Port": 80				
 	}, {
-            "Name": "Xerom Explorer Website",
+            "Name": "Main Xerom Explorer Website",
             "Address": "explorer.xerom.org",
             "Port": 80		
 	}, {
-	    "Name": "Xerom Wallet Website",
+	    "Name": "Main Xerom Wallet Website",
             "Address": "wallet.xerom.org",
             "Port": 80	
-	}	
+	}, {
+	    "Name": "Ether1.cloud Team Website",
+            "Address": "ether1.cloud",
+            "Port": 80
+	}, {
+	    "Name": "ethoFS.cloud Team Website",
+            "Address": "ethofs.cloud",
+            "Port": 80
+	}, {
+	    "Name": "Main Ether-1 RPC Server",
+            "Address": "rpc.ether1.org",
+            "Port": 80
+	}, {
+            "Name": "Ether1.cloud Team RPC Server",
+            "Address": "rpc.ether1.cloud",
+            "Port": 80			
+	}, {
+            "Name": "Ether1.cloud Team ETHO Mining Pool",
+            "Address": "ethopool.ether1.cloud",
+            "Port": 1444
+	}, {
+	    "Name": "Ether1.cloud Team XERO Mining Pool",
```

Comment:
Please remove this pool. We do not plan on integrating other pools since there are constantly new pools and pools that shutdown it is too much to maintain especially since this bot alerts us when one of the services is offline. It'd become quite overwhelming to deal with pools and unfair to the other pool ops.

Reply:
Removed pool.  

The though process was other pool ops can choose to open a full request like Dylie instructed.